recipe-server/normandy/recipes/tests/api/v1/test_api.py

```diff
@@ -1,4 +1,5 @@
 import hashlib
+from datetime import datetime, timedelta
```

Comment:
nit: Why not just importing the module like the line above?

Reply:
`datetime.datetime` is awkward, and the names `datetime` and `timedelta` are well known and unlikely to collide with other things. Also, this is the style used in the rest of the codebase.

Reply:
Consistency > personal preferences.